staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -447,6 +470,26 @@ func (a *Admission) EvaluatePodsInNamespace(ctx context.Context, namespace strin
 		if time.Now().After(deadline) {
 			return append(warnings, fmt.Sprintf("Timeout reached after checking %d pods", i+1))
 		}
+		totalEvaluatedPods++
+	}
+
+	if totalEvaluatedPods < namespaceMaxPodsToCheck {
+		for i, pod := range skippedPods {
```

Comment:
Still need to honor `namespaceMaxPodsToCheck`. You can drop the wrapping if block with this change.
```suggestion
		if remainder := namespaceMaxPodsToCheck - totalEvaluatedPods; remainder < len(skippedPods) {
			skippedPods = skippedPods[0:remainder]
		}
		for i, pod := range skippedPods {
```

Reply:
Yes, I realized this after raising the PR
I forgot to slice, And if sliced there is no need for condition
I'll make the change
Thank you